.gitignore

```
@@ -0,0 +1,6 @@
+# Default ignored files
+/shelf/
+/.idea/workspace.xml
+
+.idea
+target
```

Blue - New line here needed
```suggestion
target

```